Marlin/configuration_store.cpp

```diff
@@ -824,7 +824,13 @@ void Config_ResetDefault() {
     retract_recover_feedrate_mm_s = RETRACT_RECOVER_FEEDRATE;
   #endif
 
-  volumetric_enabled = false;
+  volumetric_enabled =
+  #if ENABLED(ENABLE_VOLUMETRIC_FILAMENT_DIAMETER)
+    true
+  #else
+    false
+  #endif
+  ;
   for (uint8_t q = 0; q < COUNT(filament_size); q++)
```

Comment:
It would be nice if you could use `ENABLED` the way @Blue-Marlin suggests, but it only works within pre-processor lines.